### PR TITLE
Add lazyloading api

### PR DESCRIPTION
**Pull request by zhantx**
I created this PR adding the api to disable lazyloading.
Sometimes it is necessary to render all the props.children (eg. server-side rendering)
Please review it:)

### CODE REVIEW

File: packages/react-swipeable-views/src/SwipeableViews.js [hunk 1]
```diff
@@ -369,6 +369,10 @@ class SwipeableViews extends Component {
      * If the computed speed is above this value, the index change.
      */
     threshold: PropTypes.number,
+    /**
+     * This is the config used to disable lazyloding
+     */
+    lazyLoading: PropTypes.bool,
```
Comment (oliviertassinari) [inline packages/react-swipeable-views/src/SwipeableViews.js]: Can we use the following wording: `disableLazyLoading`. 
The name of the boolean properties should be chosen based on the default value. We are following the HTML specification. For instance, the disabled attribute on an input element. This choice allows the shorthand notation.

File: packages/react-swipeable-views/src/SwipeableViews.js [hunk 2]
```diff
@@ -369,6 +369,10 @@ class SwipeableViews extends Component {
      * If the computed speed is above this value, the index change.
      */
     threshold: PropTypes.number,
+    /**
+     * This is the config used to disable lazyloding
```
Comment (oliviertassinari) [inline packages/react-swipeable-views/src/SwipeableViews.js]: Missing dot at the end.

File: packages/react-swipeable-views/src/SwipeableViews.js [hunk 3]
```diff
@@ -387,6 +391,7 @@ class SwipeableViews extends Component {
       delay: '0s',
     },
     resistance: false,
+    lazyLoading: true
```
Comment (oliviertassinari) [inline packages/react-swipeable-views/src/SwipeableViews.js]: `disableLazyLoading: false,`

**Comment by zhantx** [comment]
Hi Oliver, I made the changes as requested, plz review it again :)

**Comment by zhantx** [comment]
Hi @oliviertassinari , I made the changes as requested, plz review it again :)

**Comment by tafelito** [comment]
Hi, quick question regarding this. What's the point of having a flag isFirstRender=!disableLazyLoading if on componentDidMount is then reseting it to false again? 
This will prevent not to render the rest of the slides on the first render, but it will render them after the componentDIdMount change the isFirstRender to false. Is that how it's suppose to work? Am I missing something?

**Comment by oliviertassinari** [comment]
@tafelito This logic is here to improve the time to interaction metric.

**Comment by tafelito** [comment]
So it won't lazy load the other views, is that what you're saying?

**Comment by oliviertassinari** [comment]
@tafelito It's rending the first slide, **then** it's rendering the other slides. All of his occurs in two batches.

**Comment by tafelito** [comment]
ok I see 

Thanks for the clarification

**Comment by chamini2** [comment]
What if we want to not render *at all* the others unless they are shown?

**Pull request merged**